paper/ms.tex

```diff
@@ -81,46 +80,33 @@
   Evidence for accretion of massive, rocky planetary system in a sun-like pair
 }
 
-\author{
-  Semyeong Oh\altaffilmark{\pu,\lead},
-  Adrian M. Price-Whelan\altaffilmark{\pu},
-  John M. Brewer\altaffilmark{\yale},
-  Keith A. Hawkins\altaffilmark{\columbia},
-  Nathan W. C. Leigh\altaffilmark{\amnh},
-  David W. Hogg\altaffilmark{\ccpp,\mpia,\cca},
-  David N. Spergel\altaffilmark{\pu,\cca},
-  Justin Myles\altaffilmark{\yale} \etal\
-}
+\author{Semyeong Oh} % TODO: need ORCID
```

Comment:
This is the new author/affiliation style. I'm sure there's a way to clean it up / remove duplication but I haven't seen a good demo yet. Do you have an ORCID? If so, add to `\author[ORCID HERE]{Semyeong Oh}`. We'll need to get them from everyone else too.